runtime/Cpp/runtime/src/atn/TransitionType.cpp

```diff
@@ -0,0 +1,28 @@
+#include "atn/TransitionType.h"
+
+std::string antlr4::atn::transitionTypeName(TransitionType transitionType) {
+  switch (transitionType) {
+    case TransitionType::EPSILON:
+      return "EPSILON";
+    case TransitionType::RANGE:
+      return "RANGE";
+    case TransitionType::RULE:
+      return "RULE";
+    case TransitionType::PREDICATE:
+      return "PREDICATE";
+    case TransitionType::ATOM:
+      return "ATOM";
+    case TransitionType::ACTION:
+      return "ACTION";
+    case TransitionType::SET:
+      return "SET";
+    case TransitionType::NOT_SET:
+      return "NOT_SET";
+    case TransitionType::WILDCARD:
+      return "WILDCARD";
+    case TransitionType::PRECEDENCE:
+      return "PRECEDENCE";
+    default:
```

Comment:
Similar suggestion here with the default case: Don't have a default case here, but move the `return "INVALID";` to the end of the function.

Reply:
Ugh, didn't public comment. I moved it to the end, but used UNKNOWN. ATNStateType uses INVALID.